internal/api/v1/services/instance.go

```diff
@@ -99,18 +102,18 @@ func (s *InstanceService) DeleteInstance(
 		Model: gorm.Model{
 			ID: jobID,
 		},
-		Name:        name,
+		Name:        instanceName,
 		ProjectName: projectName,
 		Status:      models.JobStatusPending,
 	}
```

Comment:
[nit] BTW, we cant simply rename job to project as in some places job acts as a task like here.

Reply:
yup, leaving for your future PRs.